html-web-page/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>HTML Web Page</title>
+    <style>
+      table,
+      th,
+      td {
+        border: 1px solid black;
+      }
+
+      td:not(:first-child) {
+        text-align: center;
+      }
+      div {
+        margin: 10px;
+      }
+    </style>
+  </head>
+  <body>
+    <!-- your HTML code here 👇 -->
+    <h1><strong>Jelly of the Month Club</strong></h1>
+    <h2><strong>Make all you friends jelly of your stock of jelly</strong></h2>
```

Comment:
Headings are already bold by default, and already have a semantic meaning. No need to include `<strong>` as well